requirements.txt

```diff
@@ -1,5 +1,6 @@
 qiskit>=0.25
 pennylane>=0.15
 numpy
+sympy
```

Comment:
Usually, introducing a new dependency would be good to be considered, but in this case, Qiskit already depends on `sympy`. Worth noting that we can do this "without a cost" maybe in the PR description.

Reply:
do you mean in the changelog, or just in the PR?